src/Components/v2/__tests__/CountdownTimer.test.tsx

```diff
@@ -61,7 +61,7 @@ describe("CountdownTimer", () => {
 
       const text = timer.text()
       expect(text).toMatchInlineSnapshot(
-        `"01d 00s leftRespond by Dec 04, 8:50 AM ESTExpired offers end the negotiation process permanently."`
+        `"time remaining01d 00s leftRespond by Dec 04, 8:50 AM ESTExpired offers end the negotiation process permanently."`
```

Comment:
Why did these change?

Reply:
All of our SVG icons now include a `<title>time remaining<title>` tag. These fields have icons embeddded inline with the text.